tests/aruba/aruba.md

```diff
@@ -0,0 +1,50 @@
+# Aruba-rspec testframework for CB CLI
+The aruba test project is located in tests/aruba/
+
+## E2E tests
+
+Run without Docker:
+```
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+source /Users/mhalmy/.rvm/scripts/rvm
+gem install rspec
+gem install aruba
+gem install aruba-rspec
+gem install json
+gem install rspec-json_expectations
+gem install rspec_junit_formatter
+#Edit localvars
+source localvars
+
+rspec spec/e2e/cluster.rb  # Run a specific test
+rspec spec/e2e/*.rb # Run all tests
+rspec -f RspecJunitFormatter -o test-result.xml -f h spec/e2e/*.rb | tee test-result.html | ruby -n # Run with output files
+```
+Run with Docker locally:
+```
+#Edit localvars
+source localvars
+#run script: 
+scripts/docker-e2e-test-aruba-local.sh
+Check result in:
```

Comment:
Please create Makefile for this.

Correct command:
`. scripts/docker-e2e-test-aruba-local.sh`

Unfortunately I got error:
```
. scripts/docker-e2e-test-aruba-local.sh
-bash: scripts/docker-e2e-test-aruba-local.sh: No such file or directory
```